osu.Game.Rulesets.Mania/Mods/ManiaModHidden.cs

```diff
@@ -13,6 +15,16 @@ public class ManiaModHidden : ManiaModPlayfieldCover
         public override LocalisableString Description => @"Keys fade out before you hit them!";
         public override double ScoreMultiplier => 1;
 
+        [SettingSource("Coverage", "How much of the playfield notes should be hidden for.")]
```

Comment:
This description doesn't parse very well to me - maybe this is better?

```suggestion
        [SettingSource("Coverage", "The proportion of playfield height that notes will be hidden for.")]
```

Also would consider displaying as percentage.

Reply:
Thank you, that does sound better. I initially thought to use percentage as well, however all other mods are displayed as decimal so I think it is worth using decimal for the consistency.